src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/enums/ProtectLevel.kt

```diff
@@ -1,6 +1,6 @@
 package uk.gov.justice.digital.hmpps.hmppstier.domain.enums
 
-enum class ProtectScore {
+enum class ProtectLevel {
```

Comment:
Domain - we get assessment SCORES, which map to threshhold POINTS which returns a LEVEL